cxl/builtin.h

```diff
@@ -18,8 +18,10 @@ int cmd_check_labels(int argc, const char **argv, struct cxl_ctx *ctx);
 int cmd_identify(int argc, const char **argv, struct cxl_ctx *ctx);
 int cmd_get_supported_logs(int argc, const char **argv, struct cxl_ctx *ctx);
 int cmd_get_log(int argc, const char **argv, struct cxl_ctx *ctx);
-int cmd_get_event_interrupt_policy(int argc, const char **argv, struct cxl_ctx *ctx);
-int cmd_set_event_interrupt_policy(int argc, const char **argv, struct cxl_ctx *ctx);
+int cmd_get_event_interrupt_policy(int argc, const char **argv,
```

Comment:
Did you run clang format on this file? I think there are many lines modified and showing in diff without any code change